tensorboard/plugins/profile/tf_op_profile/tf-op-table.html

```diff
@@ -285,8 +279,16 @@
     return (node.xla && node.xla.provenance)
               ? node.xla.provenance.replace(/^.*\//, '') : "";
   },
+  _utilization: function(node) {
+    return percent(utilization(node));
+  },
+  _flameColor: function(node) {
+    return flameColor(utilization(this.node), 1, 0.2);
```

Comment:
nit: Replace `this.node` with `node` so that we use the most recent one.

Reply:
Done. Thanks!